src/collective/ifttt/actions/ifttt.py

```diff
@@ -88,25 +91,35 @@ class IftttTriggerActionExecutor(object):
     """
     timeout = 120
 
-    def __init__(self, element):
+    def __init__(self, element, context, event):
         self.element = element
+        self.context = context
+        self.event = event
 
     def __call__(self, *args, **kwargs):
-        r = self.element.request
-        ifttt_name = self.element.ifttt_name
+        Title = self.element.Title
+        URL = self.element.URL
         payload_option = self.element.payload_option
-        payload = self.element.payload
+        # TODO explore context and event; element is defined with Ifttt Trigger form
+        # TODO define ifttt_url with secret key
+        ifttt_name = self.event.title
+        secret_key = self.event.secret_key
```

Comment:
Read https://api.plone.org/ for how to read individual configuration registry setting using plone.api.

If you don't know the exact key for you configuration variable, you should be able to search it using Plone registry configuration controlpanel at your development Plone site for this add-on.